core/src/main/java/io/openepcis/epc/eventhash/EventHashGenerator.java

```diff
@@ -116,18 +114,18 @@ public static Multi<Map<String, String>> fromJson(
    *     sha3-224, sha3-256, sha3-384, sha3-512, md2, md5. using "prehash" return pre-hash strings
    * @return hash string map where key is hash algorithm and value is hash, representing each EPCIS
    *     event from InputStream
-   * @throws IOException
+   * @throws IOException reading of JSON file may throw exception
    */
   public static Multi<Map<String, String>> fromJson(
       final InputStream jsonStream, final String... hashAlgorithms) throws IOException {
-    return fromJson(jsonStream, new HashMap<String, String>(), hashAlgorithms);
+    return fromJson(jsonStream, new HashMap<>(), hashAlgorithms);
   }
 
   private static void addToContextHeader(
```

Comment:
make it non-static... 
will stop here ;-) applies to all other static methods as well